modules/__tests__/TestSequences/execSteps.js

```diff
@@ -21,6 +21,7 @@ const execSteps = (steps, history, done) => {
       if (index === steps.length)
         cleanup()
     } catch (error) {
+      console.log('caught error', error)
```

Comment:
Oh bother. That's the remnant of my debugging when I accidentally passed `done` to a `describe` call instead of an `it` call.